tests/publiccloud/run_ltp.pm

```diff
@@ -61,13 +61,12 @@ sub upload_ltp_logs
 
         my $ltp_log = Mojo::JSON::decode_json($log_file->slurp());
         my $parser = OpenQA::Parser::Format::LTP->new()->load($log_file->to_string);
+        my %ltp_log_results = map { $_->{test_fqn} => $_->{test} } @{$ltp_log->{results}};
+        my $whitelist = LTP::WhiteList->new();
 
-        if (find_whitelist_testsuite($ltp_testsuite)) {
-            my %ltp_log_results = map { $_->{test_fqn} => $_->{test} } @{$ltp_log->{results}};
-            for my $result (@{$parser->results()}) {
-                if (override_known_failures($self, {%{$self->{ltp_env}}, retval => $ltp_log_results{$result->{test_fqn}}->{retval}}, $ltp_testsuite, $result->{test_fqn})) {
```

Comment:
BTW, this part is not checking whether the test actually failed before applying whitelist. It may end up turning `pass` into `softfail`. It should be fixed in a separate PR.